packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/apple/appidpinokta.md

```diff
@@ -0,0 +1,10 @@
+In the **General Settings** section:
+
+* **Name** &mdash; Enter the name that you would expect to see on a button, such as **Sign in with Apple**.
+* **Client ID** &mdash; Paste the Services ID that you obtained from Apple when you configured the Identity Provider in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Services ID identifies the particular instance of your app and is used as the OAuth `client_id`.
+* **Client Secret Signing Key** &mdash; Click **Browse files** and locate the key file (.p8 format) that you downloaded from Apple when you created the private key for client authentcation in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>.
+* **Key ID** &mdash; Enter the Key ID that you obtained from Apple when you created the private key for client authentication in the <GuideLink link="../create-an-app-at-idp">previous section</GuideLink>. The Key ID represents the private key generated by Apple.
+* **Team ID** &mdash; Enter your Apple Team ID. This value must be the 10 character (alphanumeric) Team ID from your Apple developer account. This ID can be found in the upper-right corner of your Apple developer dashboard just below your account name.
+* **Scopes** &mdash; Leave the defaults. These scopes are included when Okta makes an OpenID Connect request to Apple.
```

Comment:
Do we want to link to scope documentation for Apple? Apple scopes are interesting because they are limited to openid, name, and email, which is very constrained in comparison to the other social idps.
cc: @venkatviswanathan-okta